src/nn/models.py

```diff
@@ -61,7 +61,35 @@ def training_step(self, batch: Tuple[torch.Tensor, torch.Tensor], batch_idx: int
         raw_loss = self.calculate_loss(y_hat, y)
         reg = self.regularization()
         loss = raw_loss + reg
-        return {'loss': loss, 'raw_loss': raw_loss.detach(), 'reg': reg.detach(), 'batch_len': x.shape[0]}
+
+        y_pred = torch.argmax(y_hat, dim=1)
```

Comment:
It should be moved to MLPClassifier or LinearClassifier classes, because otherwise it will try to calculate accuracy for UK biobank regression models